recipes/wix/bld.bat

```diff
@@ -0,0 +1,5 @@
+nuget install WiX
```

Comment:
The official ZIP binary distribution does not include a permanent URL for download. Official build instructions are here: http://wixtoolset.org/documentation/manual/v3/wixdev/building_wix.html

I have been encountering a ton of obscure build errors trying to follow these instructions though, but if someone else want to give it a go, I'm all for it :)